lib/internal/Magento/Framework/App/FeedFactory.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Framework\App;
+
+use Magento\Framework\App\FeedFactoryInterface;
+use Magento\Framework\ObjectManagerInterface;
+use Psr\Log\LoggerInterface;
+
+/**
+ * Feed factory
+ */
+class FeedFactory implements FeedFactoryInterface
+{
+    /**
+     * @var FeedProcessorInterface
+     */
+    private $feedProcessor;
+
+    /**
+     * @var LoggerInterface
+     */
+    private $logger;
+
+    /**
+     * @var ObjectManagerInterface
+     */
+    protected $objectManager;
```

Comment:
object manager has to be declared as private
Magento does not encourage to use protected accessors in newly created code